packages/ui/src/ui/pages/navigation/tabs/Queue/utils/column-builder.tsx

```diff
@@ -37,19 +39,40 @@ export function datetime<T>(name: string, getter: (row: T) => string | null): Co
     };
 }
 
-export function error<T>(name: string, getter: (row: T) => YTError | undefined): Column<T> {
+export function error<T>(
+    name: string,
+    getter: (row: T) => YTError | undefined,
+    {className}: {className?: string} = {},
```

Comment:
Are you sure you need an object here? I don't see any other parameters other than `className`. Are there any other parameters planned in the future? 

I offer 2 possible solutions:
1) Move all attributes to a single object: `{name, getter, className}: {name: string,.....}`, so you can flexibly add new parameters without editing places where this function is already used.
2) Extract className from the object: `(name: string, ...., className?: string)`. If no new parameters are planned, then we don't need the object there either.